src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2829,11 +2829,12 @@ because xbuild doesn't support framework reference assemblies.
     <_StartupAotProfile Condition="'%(ResolvedAssemblies.Filename)' == 'Xamarin.Forms.Platform.Android'">startup-xf.aotprofile</_StartupAotProfile>
     <_StartupAotProfile Condition="'$(_StartupAotProfile)' == ''">startup.aotprofile</_StartupAotProfile>
   </PropertyGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(AndroidUseDefaultAotProfile)' != 'False'">
     <AndroidAotProfile Include="$(MSBuildThisFileDirectory)$(_StartupAotProfile)" />
   </ItemGroup>
   <ItemGroup Condition="'$(AndroidEnableProfiledAot)' == 'True'">
     <_AotProfiles Include="@(AndroidAotProfile)" />
+    <_AotProfiles Include="$(AndroidAotProfiles.Split(':'))" />
```

Comment:
Splitting on `:` is going to be *really* problematic on Windows, as `:` is the *drive separator*.

Is there an actual benefit/use case to having an `$(AndroidAotProfiles)` MSBuild property, instead of using the `@(AndroidAotProfile)` build action?

If we want to keep `$(AndroidAotProfiles)`, then it should probably be `;`-separated, which in turn is *highly* problematic (because it can't be used on the command-line).

Reply:
The benefit is to be able to specify it from command line. One cannot specify items on command line when calling `msbuild` AFAIK.

I used `:` as we use it for ABIs. Now I noticed it is only used in our build system and it would be indeed problematic on Windows. The `;` can be used on command line, but needs to be escaped. So it might be still better choice.

Finally, I have found out we now have `SplitProperty` task, which looks like the best choice. It uses `;` and `,` as separators and is already used for `AndroidSupportedAbis`.